packages/office-ui-fabric-react/src/components/Button/__snapshots__/Button.deprecated.test.tsx.snap

```diff
@@ -440,370 +139,11 @@ exports[`Button renders CompoundButton correctly 1`] = `
               font-weight: 400;
               line-height: 100%;
             }
-        id="id__14"
+        id="id__1"
```

Comment:
Might not be a bad idea to put resetIds() calls in here in beforeEach block to prevent this too